tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "target": "es2018",
     "module": "CommonJS",
+    "moduleResolution": "NodeNext",
```

Comment:
@purpshell hi! Do you remember why you removed it?

Reply:
Because it was causing some issues in compilation. This whole part of the project will get a huge revamp soon